WordPress-Core/ruleset.xml

```diff
@@ -150,6 +150,9 @@
 		<!-- Covers rule: When type casting, do it like so: $foo = (boolean) $bar; -->
 		<rule ref="Generic.Formatting.SpaceAfterCast"/>
 		<rule ref="WordPress.WhiteSpace.CastStructureSpacing"/>
+		<rule ref="WordPress.WhiteSpace.CastStructureSpacing.ContainsWhiteSpace">
+			<type>warning</type>
```

Comment:
Why does this need to be a warning instead of an error? Is it not called for by WordPress's standards? I suggest that we let it be an error, since as I understand it it already is right now, no?